tests/unit/test_provider.py

```diff
@@ -122,7 +121,7 @@ def test_recid_provider_exception(db):
 
 
 def test_minting_recid(db):
-    """Test reminting doi for published record."""
+    """Test reminting recid for published record."""
```

Comment:
I think instead of checking `The case of reminting a recid for a published record`  I would transform the test to `Remint a doi for published record` and change the calls as needed.

Reply:
I think this is testing: `If there's no recid in the data it should fail `
https://github.com/CERNDocumentServer/cds-videos/blob/main/cds/modules/records/minters.py#L72
Maybe this can stay and we test the `Remint doi` in another function

Reply:
Yes I am fine with testing it in a different test!